README.md

```diff
@@ -1 +1,2 @@
-# Enumerable_methods
\ No newline at end of file
+# Enumerable_methods
```

Comment:
You README should be more descriptive.
Explain what the project is: your custom implementation of Ruby Enumerable methods etc
You can add what version of Ruby you used or what your approach was... things like that

Also add your social media links, particularly your GitHub to link back to other repos you have.

Reply:
You still have not addressed README.md recommendations, please do